src/user/dto/update-user.dto.ts

```
@@ -0,0 +1,10 @@
+import { PartialType } from '@nestjs/swagger';
+import { CreateUserDto } from './create-user.dto';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty } from 'class-validator';
+
+export class UpdateUserDto extends PartialType(CreateUserDto) {
+    @ApiProperty({ example: 'This is username' })
```

ESLint

[error] Delete `··`

(prettier/prettier)